quinn/src/lib.rs

```diff
@@ -1599,3 +1599,137 @@ impl<T: Read> Future for ReadToEnd<T> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
It'd be nice for this to live in a separate file, I think.

Reply:
Agreed.